src/test/kotlin/org/rust/lang/RustTestCaseBase.kt

```diff
@@ -68,6 +68,16 @@ abstract class RustTestCaseBase : LightPlatformCodeInsightFixtureTestCase(), Rus
             camelCaseName.split("(?=[A-Z])".toRegex())
                 .map { it.toLowerCase() }
                 .joinToString("_")
+
+        @JvmStatic
+        fun rustSources(): VirtualFile {
+            val sdkArchiveFile = LocalFileSystem.getInstance()
+                .findFileByPath("${RustTestCase.testResourcesPath}/rustc-src.zip")
+
+            return checkNotNull(sdkArchiveFile?.let { JarFileSystem.getInstance().getJarRootForLocalFile(it) }) {
+                "Rust sources archive not found. Run `./gradlew test` to download the archive."
+            }
```

Comment:
Pretty much the same thing is also done inside `WithStdlibRustProjectDescriptor#testCargoProject`